python/src/trezorlib/client.py

```
@@ -142,6 +168,10 @@ def close(self) -> None:
 
     def cancel(self) -> None:
         """Send a Cancel signal to the device, interrupting the current workflow."""
+        warnings.warn(
```

Ruff (0.15.17)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)